### PR TITLE
Add WASM Stats and Metadata Exchange Plugins to ProxyV2 Docker container

### PR DESCRIPTION
**Pull request by gargnupur**
Please provide a description for what this PR is for.
Script to add WASM binaries in Proxy V2 Container

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[x ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: pilot/docker/Dockerfile.proxyv2 [hunk 1]
```diff
@@ -11,6 +11,9 @@ FROM docker.io/istio/base:${BASE_VERSION} as default
 # obtained from debian ca-certs deb using fetch_cacerts.sh
 ADD ca-certificates.tgz /
 
+# Add WASM plugins for stats and metadata_exchange
+ADD wasm /
```
Comment (mandarjog) [inline pilot/docker/Dockerfile.proxyv2]: what is the resulting path in the container?
`/wasm/stats_1_3_1.wasm` ?

Comment (gargnupur) [reply pilot/docker/Dockerfile.proxyv2]: yup : /wasm/stats_1_3_1.wasm

Comment (mandarjog) [reply pilot/docker/Dockerfile.proxyv2]: Can you load the plugin into 1.3.1 ?

@jplevyak is this .wasm file compatible with what is in 1.3 branch or master only ?
@kyessenov any breaking changes?

Comment (kyessenov) [reply pilot/docker/Dockerfile.proxyv2]: Master has breaking changes in envoy-wasm. I am not sure which SHA was used to cut-off 1.3.1.

Comment (rlenglet) [reply pilot/docker/Dockerfile.proxyv2]: 1.3.1 will use the current HEADs of the `release-1.3` branches.

**Comment by gargnupur** [comment]
Created https://github.com/istio/istio/pull/17397 in release 1.3 directly.. abandoning this PR

**Pull request closed without merging**